sdk/core/allocator/alloc.h

```diff
@@ -891,6 +902,46 @@ class MState
 	size_t heapFreeSize;
 	size_t heapQuarantineSize;
 
+	/**
+	 * The number of entries currently in the `hazardQuarantine` array.
+	 */
+	size_t hazardQuarantineSize = 0;
+
+	/**
+	 * Returns true if there are no objects in the `hazardQuarantine` array.
+	 */
+	bool hazard_quarantine_is_empty()
+	{
+		return hazardQuarantineSize == 0;
+	}
+
+	/**
+	 * Returns true of `ptr` is already in the quarantine list.
+	 */
+	bool hazard_quarantine_contains(Capability<void> ptr)
```

Comment:
There don't seem to be any calls to this function? I was a bit worried about the `is_subset_of`.